CodeforAustralia/council-emissions-calculator
# Change icon color white when  selected.

## Overview of the Pull Request:
_Describe your changes here..._

Change made to travelMethod Icon colour  when selected it will be same as the text colour which is white. (THIS PR IS READY TO REVIEW TAKING IN ALL THE NEW CHANGES FROM MAIN.) 
This will now only focus on the icon colour to be white when active and selected.

(https://trello.com/c/z50v3GrA/297-dev-travel-method-selection-page)

## How Has This Been Tested?
- Test locally as I select the travelMethod button the icons colour have to be the same as the font colour.
- Confirm the colour of icon change when selected align with text color.
- Confirm when on hover it has to be light colour.
- Confirm after deselect the button will show light blue which is on hover. 
- Confirm the on active button will not change colour when the mouse hover. 

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [ x] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [ x] Is your pull request up-to-date with `main` branch?
- [ x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [ x] Have you written instructions on how to test that your changes work as expected?
- [x ] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?
